spec/requests/admin/teachers/show_spec.rb

```diff
@@ -33,6 +33,11 @@
         expect(response.body).to include(CGI.escape_html(Teachers::Name.new(teacher).full_name))
       end
 
+      it "links to teacher timeline" do
+        get admin_teacher_path(teacher)
+        expect(response.body).to include(admin_teacher_timeline_path(teacher))
+      end
```

Comment:
I think this should be in the view spec rather than the request one. Probably the 'display teacher info' one above too.

Reply:
I logged that as a separate issue @peterdavidhamilton (linked below) so feel free to merge this - that can be addressed whenever.

Reply:
Ah nice, I'll close. Super efficient